src/utility.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from "react";
```

Comment:
I think you should decouple React logic from this utility method! Following the Single Responsibility Principle, I would have this helper method return either directly an array of `labs`/`candidates`, or a Promise that'll resolve to the said array!

If using a Promise, I'd handle the Promise resolve in the component itself via `useState` and `useEffect`!